VirtualFilesystemTestTrait.php

```diff
@@ -63,9 +63,21 @@ trait VirtualFilesystemTestTrait {
 	public function init() {
 		if ( empty( $this->config ) ) {
 			$this->loadConfig();
+
+			if ( ! isset( $this->config['vfs_dir'] ) || false === $this->config['vfs_dir'] ) {
+				$this->config['vfs_dir'] = '';
+			} elseif ( '' !== $this->config['vfs_dir'] ) {
+				$this->config['vfs_dir'] = rtrim( $this->config['vfs_dir'], '\//' ) . '/';
```

Comment:
This will break the existing tests.

Reply:
Current tests all add a trailing slash. This only enforces it for consistency in the future tests, this will break nothing.

Reply:
Unless the test data has it configured without the trailing slash.

In many of the tests, the paths given in the `'test_data'` do not include the root or what's configured in the test fixture's `'vfs_dir'`. Then in the test method itself, it uses the 'vfs_dir'` to complete the path for the given test itself.

By overriding the 'vfs_dir'` configuration, we could break tests.

Reply:
Then let's just remove this `elseif`.

Reply:
Done in https://github.com/wp-media/phpunit/pull/21/commits/fcd152a41fac2dafa9d82bbec8a52ea495701d76.